### PR TITLE
Updating UI on comparison pages

### PR DESCRIPTION
**Pull request by teallarson**

### CODE REVIEW

**Comment by teallarson** [comment]
circleCI failure is because circleCI doesn't like my test.todos

**Comment by teallarson** [comment]
TODO:  
* taking a second look at testing this afternoon with @evantahler  (I think it could be refactored, though it does work in its current state)
* fix spacing on cells with two items (eg: with checkmark AND text)
* fix section dividers on 1920x1080 + screens

**Comment by teallarson** [comment]
Will push as-is for now.  

TODO's:  
- I don't like the current look of boxes with a ✅ AND text in the table
- weird banner appearance on 1920x1080+ screens
- would like to make the test suite for these pages more efficient.  
 
BUT all is running, so I'm thinking it can go up as is and be improved as we go.

File: components/comparisons/comparisonRow.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import Image from "../Image";
+
+export default function ComparisonRow({ rowData }) {
+  // rowData = {
+  // feature: featureName,
+  // grouparoo: true,
+  // comp: false
+  // }
```
Comment (evantahler) [inline components/comparisons/comparisonRow.tsx:8]: Maybe remove these comments

File: components/comparisons/comparisonTable.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import ComparisonRow from "./comparisonRow";
+
+export default function ComparisonTable({ comparisonData }) {
+  // comparisonData = [{
+  // category: [{
+  //   feature: featureName,
+  //   grouparoo: true,
+  //   comp: false
+  // },]
+  // }]
```
Comment (evantahler) [inline components/comparisons/comparisonTable.tsx:12]: If you did want to keep this information around, you could make a typescript type, and then say that the params you were getting into the component were of that type:

```ts
import {ComparisonData} from '../utils/comparisonPages'

export default function ComparisonTable({ comparisonData } : {comparisonData: ComparisonData}) {
  // ...
}

```

File: pages/solutions/[id].tsx [hunk 3]
```diff
@@ -2,12 +2,15 @@ import React from "react";
...
+import FeatureBanner from "../../components/comparisons/featureBanner";
 import hydrate from "next-mdx-remote/hydrate";
 import { useRouter } from "next/router";
+import { PageNavigation } from "../../components/pageNavigation";
 import { ComparisonInfo, getComparison } from "../../utils/comparisonPages";
 import { arrayToReadableList, possessiveNoun } from "../../utils/inflectors";
-
-const components = {};
+import * as components from "../../components/comparisons";
+// import { TableOfContents } from "../../components/docs/tableOfContents";
```
Comment (evantahler) [inline pages/solutions/[id].tsx:14]: More un-used comments

**Review by evantahler — APPROVED** [review]
Looks good!

Some comments below for cleanup and a suggestion that you could type some of your inputs for clarity if you wanted.

**Review by andyjih — APPROVED** [review]
👍

File: components/comparisons/comparisonTable.tsx (hunk 2 above)
Comment (tealjulia) [reply components/comparisons/comparisonTable.tsx:12]: As a workflow question.... I agree these changes should be made, but you and Andy approved it as is... do I make the changes then squash and merge?  OR just squash and merge and tidy those up in the next iteration (low-stakes here as it's just removing unneeded comments...)

**Pull request merged**